charts/galoy/values.yaml

```diff
@@ -65,7 +65,7 @@ galoy:
       repository: us.gcr.io/galoy-org/galoy-consent
       ## Digest of the image
       ##
-      digest: "sha256:be5e90bffe3ae6dc6f2729266b21022802fbed803341a90515c3ed0da6d9def1" # METADATA:: repository=https://github.com/GaloyMoney/galoy;commit_ref=84f91fb;app=consent;monorepo_subdir=apps/consent;
+      digest: "sha256:5d6ebf60a7c64aa42ccb1dd346b170be22ca5d18f02d44b249a848ba3a0ea9e4" # METADATA:: repository=https://github.com/GaloyMoney/galoy;commit_ref=d2f6f35;app=consent;monorepo_subdir=apps/consent;
```

Comment:
What is this change?

Reply:
not sure why and how it showed up. reverting the change